pages/getting-started/content/getting-started-code.raw.js

```diff
@@ -1,31 +1,130 @@
 // normally you'll start out with something like:
 // import glamorous from 'glamorous'
 
+const mediaQueries = {
+  phone: '@media only screen and (max-width: 480px)',
+}
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
I think these line breaks are added to align the coded with the text on the right, but this doesn't align well on my screen.
Maybe it would be better to have a scrollbar in the editor?